pkg/api/types.go

```diff
@@ -608,6 +620,8 @@ type MultiStageTestConfiguration struct {
 	// Workflow is the name of the workflow to be used for this configuration. For fields defined in both
 	// the config and the workflow, the fields from the config will override what is set in Workflow.
 	Workflow *string `json:"workflow,omitempty"`
+	// Environment has the values of parameters for the steps.
+	Environment TestEnvironment `json:"env,omitempty"`
```

Comment:
Why not call these parameters?

Reply:
Parameters, schmarameters.

I see "parameters" as the abstract concept and "environment" as the implementation. Doesn't make a big difference to me other than "environment" being the name used in the design document.